src/services/etherpad/hooks/Pad.js

```diff
@@ -5,16 +5,19 @@ const { Forbidden } = require('@feathersjs/errors');
 const logger = require('../../../logger');
 const globalHooks = require('../../../hooks');
 
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`@schul-cloud/commons` import should occur before import of `../../../logger`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5660864)